tests/integ/sagemaker/workflow/test_step_decorator.py

```diff
@@ -246,6 +247,20 @@ def sum(a, b):
     )
 
     try:
+        execution, _ = create_and_execute_pipeline(
```

Comment:
nit: it's better to write a separate test case rather than mixing it with the existing test. That makes it easier to identify and update this test if needed in future